fontbe/src/orchestration.rs

```diff
@@ -0,0 +1,150 @@
+//! Helps coordinate the graph execution for BE
+
+use std::{collections::HashSet, fs, path::Path, sync::Arc};
+
+use fontdrasil::orchestration::{Acl, Cache, Work, MISSING_DATA};
+use write_fonts::FontBuilder;
+
+use crate::{error::Error, paths::Paths};
+
+// Unique identifier of work. If there are no fields work is unique.
+// Meant to be small and cheap to copy around.
```

Comment:
1: to be documentation this needs there `///`.
2: treat rust docstrings like git commits: a single short descriptive line, then (if needed) a blank line and then any additional detail, so like:

```suggestion
/// Unique identifier of work.
///
/// If there are no fields work is unique.
/// Meant to be small and cheap to copy around.
```